common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -233,15 +233,13 @@ public void set(String key, String value) {
     mProperties.put(key, value);
   }
 
-  public String get(String key, final String defaultValue) {
-    String raw = mProperties.getProperty(key, defaultValue);
-    String updated = lookup(raw);
-    LOG.debug("Get Tachyon property {} as {} with default {}", key, updated, defaultValue);
-    return updated;
-  }
-
   public String get(String key) {
-    return get(key, null);
+    if (!mProperties.containsKey(key)) {
```

Comment:
If we have the checking here, then we don't need the Preconditions checking at the call site.